pkg/apis/kops/validation/legacy.go

```diff
@@ -263,24 +263,22 @@ func ValidateCluster(c *kops.Cluster, strict bool) *field.Error {
 	// Check KubeDNS.ServerIP
 	if c.Spec.KubeDNS != nil {
 		serverIPString := c.Spec.KubeDNS.ServerIP
-		if serverIPString == "" {
-			return field.Required(fieldSpec.Child("KubeDNS", "ServerIP"), "Cluster did not have KubeDNS.ServerIP set")
-		}
-
-		dnsServiceIP := net.ParseIP(serverIPString)
-		if dnsServiceIP == nil {
-			return field.Invalid(fieldSpec.Child("KubeDNS", "ServerIP"), serverIPString, "Cluster had an invalid KubeDNS.ServerIP")
-		}
+		if serverIPString != "" {
```

Comment:
Not entirely sure why you turned this validation off!  I will send a PR to turn it back on :-)